### ISSUE TITLE
Plex unable to resume after screensaver

### ISSUE THREAD

**Issue by mswsn** [aabytt/custom-screensaver-aerial#18]
I'm having issues with resuming media in Plex after the screensaver has exited. End up with an infinite loading spinner and forced to stop the media and resume again from the menu.

Was previously running [webosbrew/custom-screensaver](https://github.com/webosbrew/custom-screensaver) and encountered no issues.

This is on a 55" LG CX. 

- WebOS 5.3.0-22 
- WebOSBrew 0.6.3
- Plex 5.3.0
- Aerial Screensaver 1.0.8

**Comment by mswsn** [comment]
Found a solution by messing around with the screensaver settings. Seems like this only occurs when the source video type is set to **4k Dolby Vision (HEVC)**, changing to **4k (HEVC)** did the trick.

**Comment by aabytt** [comment]
yeah, I already have an open [issue](https://github.com/aabytt/custom-screensaver-aerial/issues/9
) on that.
I guess the problem is that playing a dolby vision in screensaver forces the underlaying paused video to release resources from its pipeline and then it can not automatically reload and get resources back because screensaver is playing as overlay with no pushing paused video app to background. 
In the app I use for movies (not Plex) I made a plugin to force a reload of paused video when screensaver is fired.
But I do not have a general non app related solution for now.

**Comment by mswsn** [comment]
My bad, should have looked that up. To add to that though, this affected all media played in Plex, not just media with HDR/DV.

**Comment by DandeMC** [comment]
same problem

**Comment by wes1993** [comment]
Hello all,
There are some news regarding this issue?

**Comment by mattokar** [comment]
Hello. Any news on this? Switching source to non-DV source did not helped. Occurs across all players, not only plex. Jellyfin and youtube also are affected.
